client/src/services/auth.ts

```diff
@@ -53,3 +53,19 @@ export const createNewAccount = async (
 
   return errors;
 };
+
+export const handleVerifyEmail = async (
+  actionCode: string
+): Promise<string> => {
+  let error = "";
+  error = await firebase
```

Comment:
Nitpick:
On line 60, `error` is initialized as a `string`.
On line 61, `error` is assigned a value with type `Promise<string>`

my suggestion is to remove line 60 and just initialize `error` as a `Promise<string>`